packages/core/src/awsService/appBuilder/explorer/nodes/deployedNode.ts

```diff
@@ -166,7 +166,7 @@ export async function generateDeployedNode(
             createPlaceholderItem(
                 localize(
                     'AWS.appBuilder.explorerNode.unavailableDeployedResource',
-                    '[Failed to retrive deployed resource.]'
+                    '[Failed to retrieve deployed resource. Please ensure stack name, region, and account credentials are correct.]'
```

Comment:
The error message may not adequately inform the user on where to check the stack name and region.